liquid.py

```diff
@@ -22,9 +26,22 @@
 log.setLevel(LOG_LEVEL)
 
 
+class InvalidCollectionName(RuntimeError):
```

Comment:
Python already has an error class for this, ValueError. Please use that, these 10 lines of code are doing nothing special.